apps/web/src/data/ecosystem.json

```diff
@@ -3590,5 +3590,13 @@
     "imageUrl": "/images/partners/gbm-domains.png",
     "category": "consumer",
     "subcategory": "nft"
+  },
+  {
+    "name": "Token Generator",
+    "description": "Token Generator is the pioneering no-code tool that revolutionized the way of creating ERC20 tokens. Since 2018, Token Generator has been the go-to platform for thousands of projects and companies that have entered Web3. Try creating a token in less than a minute with the most trusted Smart Contract Generator for ERC20 tokens.",
```

Comment:
@vittominacori 

Thanks for the updates, can you please also remove the following superlatives:
* _the pioneering no-code tool_
* _the go-to platform_
* _the most trusted Smart Contract Generator_

Reply:
Just updated the description.

I also updated the url as I noticed that other tools point to a landing page where the BASE tool is accessible.